data/json/recipes/recipe_others.json

```
@@ -62,6 +62,18 @@
     "using": [ [ "sewing_standard", 12 ] ],
     "components": [ [ [ "rag", 6 ] ], [ [ "down_feather", 40 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "down_mattress",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_OTHER",
+    "skill_used": "tailor 6",
```

That's not a valid way to specify skill requirements, below is the correct one.
```suggestion
    "skill_used": "tailor",
    "difficulty": 6,
```